examples/SplitKernels/main.cu

```diff
@@ -314,43 +323,93 @@ void runGPU(int numParticles, double energy, int batch, const int *MCIndex_host,
       // *** ELECTRONS ***
       int numElectrons = stats->inFlight[ParticleType::Electron];
       if (numElectrons > 0) {
-        transportBlocks = (numElectrons + TransportThreads - 1) / TransportThreads;
+        transportBlocks = (numElectrons + ThreadsPerBlock - 1) / ThreadsPerBlock;
         transportBlocks = std::min(transportBlocks, MaxBlocks);
 
-        TransportElectrons<<<transportBlocks, TransportThreads, 0, electrons.stream>>>(
+        TransportElectrons<<<transportBlocks, ThreadsPerBlock, 0, electrons.stream>>>(
             electrons.tracks, electrons.queues.currentlyActive, secondaries, electrons.queues.nextActive, globalScoring,
-            scoringPerVolume);
+            scoringPerVolume, electrons.soaData);
 
         COPCORE_CUDA_CHECK(cudaEventRecord(electrons.event, electrons.stream));
-        COPCORE_CUDA_CHECK(cudaStreamWaitEvent(stream, electrons.event, 0));
+        COPCORE_CUDA_CHECK(cudaStreamWaitEvent(interactionStreams[0], electrons.event, 0));
+
+        IonizationEl<<<32, ThreadsPerBlock, 0, interactionStreams[0]>>>(
+            electrons.tracks, electrons.queues.currentlyActive, secondaries, electrons.queues.nextActive, globalScoring,
+            scoringPerVolume, electrons.soaData);
+            BremsstrahlungEl<<<128, ThreadsPerBlock, 0, electrons.stream>>>(
+            electrons.tracks, electrons.queues.currentlyActive, secondaries, electrons.queues.nextActive, globalScoring,
+            scoringPerVolume, electrons.soaData);
```

Comment:
They are kind of arbitrary, indeed. To be tuned later. It's just some numbers that will allow Bernhard to run some tests.

Reply:
But already right now, it's not a general approach for a "demonstrator"...

Reply:
I unsure what you're trying to say. I'm trying to get some code ready for Bernhard, so he gets unblocked. There should be no issue to change these numbers without affecting his work in a later commit.

Reply:
I don't mind any of these numbers for now. They can become tuning parameters and probably should be tuned. From my experience a low number of threads and a very high number of threads are generally inferior to a moderate number of threads when using grid strided loops. But this has to be figured out later and is probably even GPU specific.

Reply:
The point I'm trying to make here is that the numbers are *completely* independent of the number of particles currently in flight. But even making a guess based on those is going to be hard because, well, the processes are stochastic. For that reason, I'd propose to use `transportBlocks` here. This overestimates the number of blocks actually needed, but anything else is just tuned to a very specific scenario.